Language/Functions/Communication/Serial/readString.adoc

```diff
@@ -34,10 +34,49 @@ title: Serial.readString()
 === Returns
 A `String` read from the serial buffer
 
+
 --
 // OVERVIEW SECTION ENDS
 
 
+// HOW TO USE SECTION STARTS
+[#howtouse]
+--
+
+[float]
+=== Example Code
+Demonstrate Serial.readString()
+
+[source,arduino]
+----
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop() {
+  Serial.println("Enter data:");
+  while(Serial.available()==0); //wait for data available
+  String teststr = Serial.readString(); //read until timeout
+  teststr.trim(); // remove any \r \n whitespace at the end of the String
+  if(teststr=="red") {
```

Comment:
```suggestion
  String teststr = Serial.readString();  //read until timeout
  teststr.trim();                        // remove any \r \n whitespace at the end of the String
  if (teststr == "red") {
```

Bring code formatting into compliance with the standard Arduino code formatting style, as implemented by the default configuration of the Arduino IDE's "Auto Format" feature.